### PR TITLE
[DONE] Dirk fix gradients in IE11

### PR DESCRIPTION
**Pull request by d-weeteling**

### CODE REVIEW

File: app/components/legend/legend-directive.js [hunk 1]
```diff
@@ -76,13 +76,35 @@ angular.module('legend')
...
+    var _getBrowserType = function () {
+      var userAgent = window.navigator.userAgent;
+      var REGEX = {
+        'edge': /Edge/i,
+        'mozilla': /Mozilla/i,
+        'gecko': /Gecko/i,
+        'webkit': /Webkit/i,
+      };
+
+      if (userAgent.match(REGEX.mozilla)) {
```
Comment (RoelvandenBerg) [inline app/components/legend/legend-directive.js]: Shouldn't this be `REGEX.edge`?

**Comment by RoelvandenBerg** [comment]
It doesn't work.

**Comment by RoelvandenBerg** [comment]
This should fix https://github.com/nens/lizard-nxt/issues/2153

File: app/components/legend/legend-directive.js (hunk 1 above)
Comment (d-weeteling) [inline app/components/legend/legend-directive.js]: The useragent for Edge matches REGEX.mozilla. This was a suprise to me too, but it's official: https://msdn.microsoft.com/en-us/library/hh869301(v=vs.85).aspx

**Pull request merged**